packaging/systemd/ovs-vswitchd-microshift.conf

```diff
@@ -0,0 +1,2 @@
+[Service]
```

Comment:
Let's put microshift- as a prefix in the file name to be consistent with other files.

Reply:
👍  done.